action.yml

```diff
@@ -17,13 +17,30 @@ runs:
     # * https://github.com/JuliaLang/Pkg.jl/issues/2011
     # * https://github.com/JuliaRegistries/General/issues/16777
     # * https://github.com/JuliaPackaging/PkgServer.jl/issues/60
-  - run: julia --color=yes -e 'using Pkg; VERSION >= v"1.5-" && !isdir(joinpath(DEPOT_PATH[1], "registries", "General")) && Pkg.Registry.add("General")'
+  - run: |
+      julia --color=yes -e '
+        using Pkg
+        if VERSION >= v"1.5-" && !isdir(joinpath(DEPOT_PATH[1], "registries", "General"))
+          function f()
+              @info("Attempting to clone the General registry")
+              Pkg.Registry.add("General")
+          end
+          delays = ExponentialBackOff(; n = 10, max_delay = 120)
+          try
+              retry(f; delays = delays)()
+              @info("Successfully cloned the General registry")
+          catch ex
+              msg = "I was unable to clone the General registry. However, the build will continue."
+              @error(msg, exception=(ex,catch_backtrace()))
+          end
+        end'
     shell: bash
     env:
+      JULIA_PKG_SERVER: ""
```

Comment:
Could you move this back to where it was, please? The comment is explains why it is set.

Reply:
Done